epic-integration/src/main/kotlin/com/backend/plugins/EpicCommunication.kt

```diff
@@ -62,13 +65,33 @@ class EpicCommunication {
         return response.receive()
     }
 
+    suspend fun readPatient(patientId: String, outputFormat: String = "json"): HttpResponse {
```

Comment:
Litt generelt spørsmål: Burde vi bruke ha docstring for funksjonene? F.eks. her kunne det vært nyttig å vite hva outputFormat kan være, for det skal vel i utgangspunktet bare være "json" eller "xml".